modules/core-updates/9.0.md

```diff
@@ -178,6 +262,29 @@ If you need to integrate your code with the authentication process you should no
 - [AdminController::isAnonymousAllowed is now public](https://github.com/PrestaShop/PrestaShop/pull/35810)
 - [Login page and authentication migrated](https://github.com/PrestaShop/PrestaShop/pull/35983)
 
+### Symfony context refacto
+
+Lots of core code is based on the legacy `Context` class that stores some data as a singleton, but this legacy implementation has many drawbacks so a refactoring was started in PrestaShop 9.0 to replace this usage.
```

Comment:
```suggestion
A significant amount of code in the core relies on the legacy 'Context' class, which stores certain data as a singleton. However, this legacy implementation has many drawbacks, so refactoring was initiated in PrestaShop 9.0 to replace this usage.
```

Reply:
Ok except for `Context` that should be in backquotes IMHO, as it's a reference to some code